committee-code-of-conduct/bootstrapping-process.md

```diff
@@ -0,0 +1,36 @@
+# Bootstrapping Document for Operator Framework Code of Conduct Committee
+
+This document (created by the Operator Framework community chairs) outlines what the Code of Conduct Committee (CoCC) is responsible for, how to create it, and the rough processes for them to get started.
+
+## Objectives of the CoCC
+* Maintain the Code of Conduct (CoC) document and iterate as needed.
+  * All CoC revisions must be approved by the community chairs.
+  * Currently, we use the CNCF CoC. Any addendums or changes based off learnings in the Operator Framework community should be owned by this body of people, the Operator Framework Code of Conduct Committee (CoCC).
+* Determine and make transparent how CoC issues and incidents are reported and handled in the community.
+* Enforce the Code of Conduct within the Operator Framework community.
+  * Discuss with parties affected
+  * Come up with resolution
+  * For moderation on a day-to-day basis, work with the existing community-management subproject under SIG Contributor Experience
+* Figure out how to keep and then keep a record of CoC reports and outcomes for precedent _This is something we are still actively discussing on how to do and is not fully hardened._
+
+## Formation of the Code of Conduct Committee (CoCC):
+* The CoCC consists of 5 members. In the first election, the top 3 voted people will be appointed 2 year terms and the other 2 members will be appointed for a 1 year term.
+  * CoCC members appointed for a 1 year term may be elected again the following year.
+  * The community chairs are eligible to nominate people for the Code of Conduct Committee (CoCC). _This may change during the next election._
+  * The community chairs votes on nominees for the CoCC
+* Characteristics and Guidance for nominating people for the CoCC:
+  * Do not have to be part of the Operator Framework or CNCF community
+  * Previous experience on an Ethics Committee or Code of Conduct Committee is appreciated
+  * Has demonstrated integrity, professionalism, and positive influence within the community
+  * Experience with the tools which we use to communicate (Zoom, Slack, GitHub, etc.) within the Operator Framework community is appreciated
+  * Is generally a responsible human
```

Comment:
i think this is covered by "Has demonstrated integrity, professionalism, and positive influence within the community". 
"Is generally a responsible human"  seems vague and thus does not add much.